Cargo.toml

```diff
@@ -37,19 +38,27 @@ serde = { default-features = false, optional = true, version = "1.0" }
 serde_json = { default-features = false, optional = true, version = "1.0" }
 tokio-postgres = { default-features = false, optional = true, version = "0.7" }
 
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+wasm-bindgen = { default-features = false, version = "0.2" }
```

Comment:
It's a good idea to declare these things as optional **even with** the cfg guard, including `wasm-bindgen`/`js-sys`. Chrono found this out and reverted it: <https://github.com/chronotope/chrono/issues/1164>. There's some nasty circular dep situations that can occur otherwise.

Reply:
Good call out and thank you for raising this @robjtede 

Admittedly, I'm still getting up to speed on this so it's good to find out what not to do also!